tests/modules/damageprofiler/test.yml

```diff
@@ -1,110 +1,77 @@
+- name: damageprofiler test_damageprofiler_reference
+  command: nextflow run ./tests/modules/damageprofiler -entry test_damageprofiler_reference -c ./tests/config/nextflow.config  -c ./tests/modules/damageprofiler/nextflow.config
   tags:
     - damageprofiler
   files:
-    - path: output/damageprofiler/test/3p_freq_misincorporations.txt
-      md5sum: de3b84d946a6b63cdcfadf82bf6854c0
     - path: output/damageprofiler/test/3pGtoA_freq.txt
-      md5sum: 61c903b1504ed7d7182570dfc75e4498
+      md5sum: 56c165bbc9197e0b3d5bd3d464f9e235
+    - path: output/damageprofiler/test/3p_freq_misincorporations.txt
+      md5sum: 62e60259dd7fa2b54cdab6afc1adc9c4
     - path: output/damageprofiler/test/5pCtoT_freq.txt
-      md5sum: 15a75b60ee519b61ce04a83fe3afe855
+      md5sum: 38b5d7c87ec2d8422bc5587095711c5f
     - path: output/damageprofiler/test/5p_freq_misincorporations.txt
-      md5sum: 3b3240d6c1a3491e461b39199a9fcfe3
-    - path: output/damageprofiler/test/DamagePlot_five_prime.svg
+      md5sum: 05e041ff47e7c6d08ede51f5c81a2464
+    - path: output/damageprofiler/test/DNA_comp_genome.txt
+      md5sum: e4d949c6aceddc2c2ef14de9c4aafde5
+    - path: output/damageprofiler/test/DNA_composition_sample.txt
+      md5sum: 16be9c91a06fa7defee376230b12862f
     - path: output/damageprofiler/test/DamagePlot.pdf
+    - path: output/damageprofiler/test/DamagePlot_five_prime.svg
     - path: output/damageprofiler/test/DamagePlot_three_prime.svg
     - path: output/damageprofiler/test/DamageProfiler.log
-      contains:
-        - "FINISHED SUCCESSFULLY"
+    - path: output/damageprofiler/test/Length_plot.pdf
+    - path: output/damageprofiler/test/Length_plot_combined_data.svg
+    - path: output/damageprofiler/test/Length_plot_forward_reverse_separated.svg
     - path: output/damageprofiler/test/dmgprof.json
-      md5sum: 2e54e712d2ae9e32c4c298e5fd8f60fe
-    - path: output/damageprofiler/test/DNA_comp_genome.txt
-      md5sum: fea48af1ecf491b439d36d4a919473df
-    - path: output/damageprofiler/test/DNA_composition_sample.txt
-      md5sum: 9e17a0b1e5ad4eb13201cd24ad8507dd
+      md5sum: a011a47edaadb3a07527c329a6492570
+    - path: output/damageprofiler/test/editDistance.txt
+      md5sum: 7590dac007633ee11025cc99ec9f009b
     - path: output/damageprofiler/test/edit_distance.pdf
     - path: output/damageprofiler/test/edit_distance.svg
-    - path: output/damageprofiler/test/editDistance.txt
-      md5sum: 04d14b449a5afa8b5dbff0dfa762356b
-    - path: output/damageprofiler/test/Length_plot_combined_data.svg
-    - path: output/damageprofiler/test/Length_plot_forward_reverse_separated.svg
-    - path: output/damageprofiler/test/Length_plot.pdf
     - path: output/damageprofiler/test/lgdistribution.txt
-      md5sum: df2e19195185ea9ee05e8e84b2948f36
+      md5sum: 583fd98094de3fb462b133daeaadcf52
```

Comment:
why do the md5sums change 😱

Reply:
There is slight deterministic behaviour IIRC... that one I want to check though once the chr22 -> chr21 is in :)

Reply:
This one I mena: https://github.com/nf-core/test-datasets/pull/619